src/components/Table.tsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { useMemo } from 'react';
+import { Person } from '../types';
+
+const PersonLink = ({ person }: { person: Person }) => {
+  return (
+    <Link
+      to={`/people/${person.slug}`}
+      className={cn({ 'has-text-danger': person.sex === 'f' })}
+    >
+      {person.name}
+    </Link>
+  );
+};
+
+const Table = ({ people }: { people: Person[] }) => {
+  const param = useParams();
+
+  const computedPeople = useMemo(
+    () =>
+      people.map(el => {
+        const mother = people.find(person => person.name === el.motherName);
+        const father = people.find(person => person.name === el.fatherName);
+
+        return { ...el, mother, father };
+      }),
+    [people],
+  );
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {computedPeople.map(person => (
```

Comment:
Let's extract all necessary keys here to avoid typing `person.` each time. 
`({ name, ... }) `